cmake/CompilerFlags.cmake

```diff
@@ -152,6 +152,9 @@ function(set_default_compile_options target)
 
     add_supported_cxx_flags(${target} PRIVATE ${warning_flags})
 
+    # Support try/catching both asynchronous and synchronous exceptions
+    add_supported_cxx_flags(${target} PRIVATE /EHa)
```

Comment:
Does it work for gcc?

Reply:
I don't think it does, though I don't know what the gcc equivalent is.  I was basically trying this out in CI to see what happened.  I think I need to do more than just filter it for MSVC though.  glslang is compiled with "/EHs" (IIRC) and there's an error thrown for conflicting "/EHa" and "/EHs" flags there.

So I think I need to limit this flag to the render-test project on MSVC.

Reply:
My question was more of if gcc prints a warning that may get treated as an error; something like unrecognized argument.
I gave a quick try on my WSL, which I believe uses either gcc or clang, and I didn't see any warning or error messages.
I thought there would be but I am not sure why I didn't see any.

Reply:
Googling, I couldn't tell what SEH support there is in any version of gcc.  Sounds like maybe it's at least tolerated on some version of gcc.  On linux though, it does make g++ barf, so I'm still working on getting this option applied for only MSVC.  I don't know enough about the SEH support in gcc on windows to try to keep it enabled for non-MSVC.

Reply:
I believe SEH is a windows only thing.

Reply:
Yeah, SEH is windows only.  Not sure if that makes it MSVC-only though, though I think it is.  IIUC, we only support using MSVC on the slang windows build, so it's not an important distinction.  The only CI protected build that barfs with "/EHa" flag are the linux builds.